### ISSUE TITLE
RuntimeWarning: Can save best model only with val_acc available, skipping.   'skipping.' % (self.monitor), RuntimeWarning)

### ISSUE THREAD

**Issue by tianyunzqs** [shibing624/pycorrector#70]
在rnn_crf中，一个batch跑完后，保存模型的时候报错，跑完模型没保存，请问这是哪里的问题呢？
fit的时候有20%的训练数据作为验证数据啊

**Comment by shibing624** [comment]
跑完一个epoch保存，一个batch不保存。

**Comment by tianyunzqs** [comment]
@shibing624 对对   说错了  不好意思   是跑完一个epoch保存报错
16360/16360 [==============================] - 350s 21ms/step - loss: 36.6736 - crf_viterbi_accuracy: 0.6060 - val_loss: 36.3753 - val_crf_viterbi_accuracy: 0.8981
/usr/local/lib/python3.5/dist-packages/keras/callbacks.py:434: RuntimeWarning: Can save best model only with val_acc available, skipping.
  'skipping.' % (self.monitor), RuntimeWarning)
Epoch 2/10
2019-07-03 14:43:57,504 - __main__ - INFO - batch: 0
 1792/16360 [==>...........................] - ETA: 4:56 - loss: 35.2050 - crf_viterbi_accuracy: 0.90012019-07-03 14:44:33,942 - __main__ - INFO - batch: 28

**Comment by shibing624** [comment]
我本地又测试了，可以保存。
